client/plots/matrix.renderers.js

```diff
@@ -456,6 +456,21 @@ export function setRenderers(self) {
 			.attr('transform', `translate(${legendX},${legendY})`)
 
 		if (hc.xDendrogramHeight) {
+			//calculate the max gene label
+			let maxLabelWidth = 0,
+				maxLabelNumChars = 0
+			if (hc.xDendrogramHeight) {
```

Comment:
@congyu-lu why is this condition repeated, it's already inside the condition block for `if (hc.xDendrogramHeight)`

Reply:
deleted, thanks